src/Chargebee.ts

```diff
@@ -33,6 +33,14 @@ const ChargebeeReactNative = ChargebeeReactNativeModule
     );
 
 export default class Chargebee {
+  /**
+   * Sets up Chargebee SDK with site, API key and SDK IDs for Android and iOS.
+   * 
+   * @param {String} site Chargebee site
```

Comment:
Should we add an example here? We've seen questions like whether we should give the full URL (`sriram-test.chargebee.com`) or only the subdomain (`sriram-test`).

Reply:
@cb-sriramthiagarajan Updated examples for both.